stablehlo/integrations/python/stablehlo/savedmodel/stablehlo_to_tf_saved_model.py

```diff
@@ -208,7 +208,7 @@ def to_tf_saved_model(
 def stablehlo_to_tf_saved_model(
     module: ir.Module,
     saved_model_dir: os.PathLike,
-    target_version: str = stablehlo.get_current_version(),
+    target_version: str,
```

Comment:
WDYT about defaulting to the minimum version instead?

Reply:
That should be fine as well given [min, current] is a moving target following the compatibility window. 
Let us keep this a required field for now delegating version choice to the users.